### DISCUSSION TITLE
Apps (integrations?) cannot be installed

### DISCUSSION THREAD

**Discussion by apbarratt** [nextcloud/all-in-one#709] [Questions] [answered]
Hi guys,

Super happy to have my AIO finally running, my entire camera roll from my phone just uploaded and to a custom folder no less!  Take that dropbox!  What an excellent app the Nextcloud one is :)

Now then, I went to view the locations section of my photos page and the first thing it tells me is to install the maps integration (app? I'm a bit confused by the terminology here).

However, whenever I click the "Download and enable" button, I'm faced with a silent minute or so before a popup error appears reading "An error occured during the request. Unable to proceed"

It would seem this happens with any integration I attempt to load on the page.

I also note I cannot find a way to navigate to this, for want of a better word, app store.  I can get to the apps page for stuff already installed etc, but aside from that, I need to manually type `/integrations` on the end to see the store load up.

Andy

**Comment by szaimen** [comment]
> Super happy to have my AIO finally running, my entire camera roll from my phone just uploaded and to a custom folder no less! Take that dropbox! What an excellent app the Nextcloud one is :)

Great that you like it! :)

> Now then, I went to view the locations section of my photos page and the first thing it tells me is to install the maps integration (app? I'm a bit confused by the terminology here).

yes, the maps app is required for this...

> However, whenever I click the "Download and enable" button, I'm faced with a silent minute or so before a popup error appears reading "An error occured during the request. Unable to proceed"

I fear this is probably an issue of the maps app. Do you have an error in the Nextcloud logs?

I just tested to enalbe the maps app on my test instance and it works for me...

> It would seem this happens with any integration I attempt to load on the page.
>
> I also note I cannot find a way to navigate to this, for want of a better word, app store. I can get to the apps page for stuff already installed etc, but aside from that, I need to manually type `/integrations` on the end to see the store load up.

can you show a screenshot of this?

**Comment by szaimen** [comment, marked as answer]
> However, whenever I click the "Download and enable" button, I'm faced with a silent minute or so before a popup error appears reading "An error occured during the request. Unable to proceed"

I just found the reason for this: https://github.com/nextcloud/all-in-one/issues/715

**Reply by szaimen** [reply]
See https://github.com/nextcloud/all-in-one/issues/715#issuecomment-1136268282 how to fix it
